resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 500.00,
+          "fundsDeductedDuringWithdrawal": true,
+          "withdrawalReason": "Regular withdrawal",
+          "automaticRecoveryAmount": 250.00
+        }
+        </pre>
+                    </td>
+                    <td>
+                        <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>The accept header is invalid</p>
+                        <p class="code--block">
+                            <strong>lisaManagerReferenceNumber:</strong><br>
+                            <a href="#section/Testing">Use your test user profile</a><br>
+                            <br>
+                            <strong>accountId:</strong><br>1234567890
+                            <br>
+                            <br>
+                            <strong>Accept:</strong><br> application/vnd.hmrc.1.0
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "claimPeriodStartDate": "2017-12-06",
+          "claimPeriodEndDate": "2018-01-05",
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 500.00,
+          "fundsDeductedDuringWithdrawal": true,
+          "withdrawalReason": "Regular withdrawal",
+          "automaticRecoveryAmount": 250.00
+        }
+        </pre>
+                    </td>
+                    <td>
+                        <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>Withdrawal charge has already been reported</p>
+                        <p class="code--block">
+                            <strong>lisaManagerReferenceNumber:</strong><br>
+                            <a href="#section/Testing">Use your test user profile</a><br>
+                            <br>
+                            <strong>accountId:</strong><br>0000000409
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "claimPeriodStartDate": "2017-12-06",
+          "claimPeriodEndDate": "2018-01-05",
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 
```

Comment:
This does not require a Content-Type header (the RAML is incorrect) as this is a retrieval endpoint.

<img width="659" alt="Screenshot 2023-02-10 at 09 04 41" src="https://user-images.githubusercontent.com/71127865/219022990-00a0455b-c306-4d81-826a-73f26851356c.png">

Please correct this in this PR by removing this reference as discussed. Thanks.

<img width="1477" alt="Screenshot 2023-02-15 at 11 59 08" src="https://user-images.githubusercontent.com/71127865/219023457-2e9e1f9c-c537-4ab5-b5cc-43117b1e1cec.png">